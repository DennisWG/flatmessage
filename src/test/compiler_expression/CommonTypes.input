module Playground.Net.CommonTypes;

data Vector2d
{
    float x;
    float y;
}

data Vector3d
{
    float x;
    float y;
    float z;
}