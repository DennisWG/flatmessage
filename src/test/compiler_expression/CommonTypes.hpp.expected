namespace Playground::Net
{
#pragma pack(push, 1)

    struct Vector2d
    {
    public:
        float x;
        float y;
    };

    struct Vector3d
    {
    public:
        float x;
        float y;
        float z;
    };

#pragma pop
}
